dockers/actions-plan-preview/planpreview.go

```diff
@@ -170,6 +184,15 @@ func makeCommentBody(event *githubEvent, r *PlanPreviewResult) string {
 		if app.ApplicationKind == "TERRAFORM" {
 			lang = "hcl"
 		}
+
+		l := utf8.RuneCountInString(app.PlanDetails)
+		if detailLen+int64(l) > reservedDetailMessagesLen {
+			fmt.Fprint(&b, detailsOmittedMessage)
```

Comment:
We have to use `fmt.Fprintf(&b, detailsFormat, lang, detailsOmittedMessage)` to ensure that the message is wrapped in the `Details` block.

Reply:
I think it would be redundant to click `Details` block and just see ignored message, so I left out the details tag.
How do you feel about this?

Reply:
Thanks for your explanation. Nice point but I feel that we should wrap it inside the `Details` block to keep the comment layout.

Reply:
It's certainly more consistent that way.
I will change the message to be wrapped inside the Detail block.
Thank you!

Reply:
I fixed this in [ae8163e](https://github.com/pipe-cd/pipecd/pull/3253/commits/ae8163e26c2024aa77c471c6af48e960ccaa4f5c). PTAL